ncl/ncl_entries/sin_trigonometry.ipynb

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
<p>I think we can include actual math formatting here, I think that may work better than a code block.</p><p> </p><p>Although, I think it might be okay to assume that we don't need to explain the math behind sine?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>